raft/src/main/java/org/apache/kafka/raft/QuorumState.java

```diff
@@ -32,25 +32,25 @@
  * This class is responsible for managing the current state of this node and ensuring only
  * valid state transitions.
  *
- * Unattached =>
+ * Unattached transitions to:
```

Comment:
Changes to this file are unrelated to this PR. Made them here so that `docsJar` would succeed.